aiidalab_widgets_base/structures.py

```diff
@@ -688,9 +694,13 @@ def _pybel_opt(self, smiles, steps):
         pbmol.localopt(forcefield="mmff94", steps=100)
```

Comment:
@cpignedoli could you please comment on this code? It seems we are first optimizing with UFF, then gaff, then mmff94, and finally UFF again?

Reply:
also here @pgasparo should comment